### ISSUE TITLE
$EnableSSLWarning Param Not Used In New-StatusCakeHelperTest

### ISSUE THREAD

**Issue by hukdizzle** [Oliver-Lii/statuscake-helpers#33]
Hi Oliver,

It appears that the $EnableSSLWarning parameter for New-StatusCakeHelperTest is not implemented within the process block of the function.

**Comment by Oliver-Lii** [comment]
Hi hukdizzle,

I'm not quite sure I follow. 

The $EnableSSLWarning parameter within New-StatusCakeHelperTest is utilized in the following part of the function as long as a value is supplied.

```powershell
    $ParameterList = (Get-Command -Name $MyInvocation.InvocationName).Parameters
    $ParamsToIgnore = @("baseTestURL","Username","ApiKey")
    foreach ($key in $ParameterList.keys)
    {
        $var = Get-Variable -Name $key -ErrorAction SilentlyContinue;
        if($ParamsToIgnore -contains $var.Name)
        {
            continue
        }
        elseif($var.value -or $var.value -eq 0)
        {   #Validate Range accepts $true or $false values as 0 or 1 so explictly convert to int for StatusCake API        
            $psParams.Add($var.name,$var.value)                  
        }
    }

    $statusCakeAPIParams = $psParams | ConvertTo-StatusCakeHelperAPIParams
```

**Comment by hukdizzle** [comment]
Ah my apologies I totally missed that! Does the $EnableSSLWarning parameter correlate to the "Validate SSL" setting within an HTTPS test? I have tried providing both $true and 1 to that parameter and it does not seem to have an effect on that setting within the Statuscake UI of a test.

**Comment by Oliver-Lii** [comment]
I'm not certain as I only have access to a Community plan and don't have access to that feature in the UI.

If you change the "Validate SSL" setting on a test in the UI and use the function Get-StatusCakeHelperDetailedTestData on the test. Do you see if the $EnableSSLWarning value in the response is changed to match?

**Comment by Oliver-Lii** [comment]
I've tested this and checked using Get-StatusCakeHelperDetailedTestData that the "Validate SSL" setting in the UI does correlate to the $EnableSSLWarning parameter. 

I believe this may be a bug with the StatusCake API. Changing the setting via the API and it reports that it is successfully changed. Attempting to change the setting again to the same value through the API and it reports no data has been changed.

Using Get-StatusCakeHelperDetailedTestData shows that the EnableSSLWarning value hasn't changed so it appears something is not being set properly when it's modified via the API.
